packages/dhis2/src/Adaptor.js

```diff
@@ -107,7 +110,10 @@ axios.interceptors.response.use(
 
     const details = error.response?.data;
 
-    Log.error(error.message || "That didn't work.");
+    console.error(
+      `✗ Error at ${new Date()}:\n∟`,
```

Comment:
isn't the idea that the runtime prints the data already? (or decides if to print the date?)

Reply:
Just to be clear the _date_ is redundant, we should just log the message please

Reply:
Just to be clear the _date_ is redundant, we should just log the message please